PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/webapp/WebAppService.java

```diff
@@ -63,7 +64,7 @@ public static WebAppSettingModel convertConfig2Settings(final WebAppConfig confi
         final WebAppSettingModel settings = new WebAppSettingModel();
         settings.setSubscriptionId(config.getSubscription().subscriptionId());
         // creating if id is empty
-        settings.setCreatingResGrp(StringUtils.isEmpty(config.getResourceGroup().id()));
+        settings.setCreatingResGrp(config.getResourceGroup() instanceof Draft || StringUtils.isEmpty(config.getResourceGroup().id()));
```

Comment:
what's the case when config is not a draft and doesn't have id?

Reply:
e.g. developer make another class inherited from `ResourceGroup`.